testsuite/do_tests.sh

```diff
@@ -131,6 +131,10 @@ NEST="nest_serial"
 
 HAVE_MPI="$(sli -c 'statusdict/have_mpi :: =only')"
 
+if test "${HAVE_MPI}" = "true"; then
+  MPIRUN="$(command -v mpirun)"
```

Comment:
For MPICH, this may have to be `mpiexec`, but the user should have this defined in `~/.nestrc`. Maybe it is better to use something like this?
```suggestion
MPIRUN="$(sli -c '1 () () mpirun cst 0 get =only')"
MPIRUN="$(command -v $MPIRUN)"
```

Reply:
You are right. We should actually use this to build the full command further down as well.